lib/wickedbase.pm

```diff
@@ -562,13 +561,24 @@ sub post_run {
     $self->{wicked_post_run} = 1;
 
     $self->do_barrier('post_run');
+    if ($self->{name} ne 'before_test' && get_var('WICKED_TCPDUMP')) {
+        script_run('kill ' . get_var('WICKED_MYPID'));
+        select_console('root-virtio-terminal1') if (get_var('VIRTIO_CONSOLE_NUM', 1) > 1);
+        upload_file('/tmp/tcpdump' . $self->{name} . '.pcap', 'tcpdump' . $self->{name} . '.pcap');
```

Comment:
Shouldn't we select the fist serial console here.
I know it's done in upload_wicked_logs() somehow, but I think we should make this thing isolated.

Reply:
can you please come up with code suggestion , because I didn't get I thought this way to go to use offline upload , no ?

Reply:
It does. But you are then on console "root-virtio-terminal1" which you might not want for the next commands.
Just call `$self->select_serial_terminal();` after `upload_file()`

Reply:
ah I see now . we need to do it as one line call somehow ... but outside this PR for sure